helpers/QueryHelper.php

```diff
@@ -214,6 +214,35 @@ public static function insert_multiple_rows( $table, $request ) {
 		return true;
 	}
 
+	/**
+	 * Make tge where clause base on its column operator and values.
+	 *
+	 * If the operator is IN then make the clause like `WHERE column_name IN (value1, value2, ...)`
+	 * Otherwise the clause would be `WHERE column_name = 'value'`
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param array $where 	The where clause array. e.g. array( 'id', 'IN', array(1, 2, 3) ) or array( 'id', '=', 1 )
+	 *
+	 * @return string
+	 */
+	private static function make_clause( array $where ) {
+		list ( $field, $operator, $value ) = $where;
+
+		if ( strtoupper($operator) === 'IN' ) {
+			$value = array_map(
```

Comment:
If operator is `IN`, we can use simply
`$value = self::prepare_in_clause( $value );`

Instead of array_map and imploding values